python/tests/test_runbolt_dev.py

```
@@ -157,6 +220,111 @@ def fake_run_dev_reloader(command, watch_paths, ignore_dir_names, ignore_paths,
+    file itself — editing a helper module api.py imports would never reload.
+    """
+    (tmp_path / "urls.py").write_text("urlpatterns = []\n")
+    (tmp_path / "api.py").write_text(
+        "from django_bolt.api import BoltAPI\nimport dev_watch_helper_a\napi = BoltAPI()\n"
+    )
+    helper = tmp_path / "dev_watch_helper_a.py"
+    helper.write_text("VALUE = 1\n")
+
+    monkeypatch.syspath_prepend(str(tmp_path))
+    _clear_modules(monkeypatch, "urls", "api", "dev_watch_helper_a")
+    monkeypatch.setattr(runbolt_module, "apps", SimpleNamespace(ready=True, get_app_configs=lambda: []))
```

Ruff (0.15.21)

[warning] Prefer `list` over useless lambda

Replace with `lambda` with `list`

(PIE807)